python/cudf/cudf/core/series.py

```diff
@@ -3630,7 +3630,7 @@ def cov(self, other, min_periods=None):
 
         return lhs._column.cov(rhs._column)
 
-    def corr(self, other, method="pearson", min_periods=None):
+    def corr(self, other, method, min_periods=None):
```

Comment:
```suggestion
    def corr(self, other, method='pearson', min_periods=None):
```
according to https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.corr.html